libevm/Instruction.cpp

```diff
@@ -24,204 +24,204 @@ namespace eth
+	{ Instruction::XSHR,         { "XSHR",                2,    1,  Tier::Special } },
+	{ Instruction::XSAR,         { "XSAR",                2,    1,  Tier::Special } },
+	{ Instruction::XROL,         { "XROL",                2,    1,  Tier::Special } },
+	{ Instruction::XROR,         { "XROR",                2,    1,  Tier::Special } },
+	{ Instruction::XPUSH,        { "XPUSH",               1,    1,  Tier::VeryLow } },
+	{ Instruction::XMLOAD,       { "XMLOAD",              1,    1,  Tier::VeryLow } },
+	{ Instruction::XMSTORE,      { "XMSTORE",             2,    0,  Tier::VeryLow } },
+	{ Instruction::XSLOAD,       { "XSLOAD",              1,    1,  Tier::Special } },
+	{ Instruction::XSSTORE,      { "XSSTORE",             2,    0,  Tier::Special } },
+	{ Instruction::XVTOWIDE,     { "XVTOWIDE",            1,    1,  Tier::VeryLow } },
+	{ Instruction::XWIDETOV,     { "XWIDETOV",            1,    1,  Tier::VeryLow } },
+	{ Instruction::XPUT,         { "XPUT",                3,    1,  Tier::Special } },
+	{ Instruction::XGET,         { "XGET",                2,    1,  Tier::Special } },
+	{ Instruction::XSWIZZLE,     { "XSWIZZLE",            2,    1,  Tier::Special } },
+	{ Instruction::XSHUFFLE,     { "XSHUFFLE",            3,    1,  Tier::Special } },
+	{ Instruction::CREATE,       { "CREATE",              3,    1,  Tier::Special } },
+	{ Instruction::CREATE2,      { "CREATE2",             4,    1,  Tier::Special } },
+	{ Instruction::CALL,         { "CALL",                7,    1,  Tier::Special } },
+	{ Instruction::CALLCODE,     { "CALLCODE",            7,    1,  Tier::Special } },
+	{ Instruction::RETURN,       { "RETURN",              2,    0,  Tier::Zero } },
+	{ Instruction::STATICCALL,   { "STATICCALL",          6,    1,  Tier::Special } },
+	{ Instruction::DELEGATECALL, { "DELEGATECALL",        6,    1,  Tier::Special } },
+	{ Instruction::REVERT,       { "REVERT",              2,    0,  Tier::Special } },
+	{ Instruction::INVALID,      { "INVALID",             0,    0,  Tier::Zero    } },
+	{ Instruction::SUICIDE,      { "SUICIDE",             1,    0,  Tier::Special } },
  
 	// these are generated by the interpreter - should never be in user code
-	{ Instruction::PUSHC,        { "PUSHC",          3,     0,    1, Tier::VeryLow } },
-	{ Instruction::JUMPC,        { "JUMPC",          0,     1,    0, Tier::Mid } },
-	{ Instruction::JUMPCI,       { "JUMPCI",         0,     2,    0, Tier::High } },
+	{ Instruction::PUSHC,        { "PUSHC",               0,    1, Tier::VeryLow } },
+	{ Instruction::JUMPC,        { "JUMPC",               1,    0, Tier::Mid } },
+	{ Instruction::JUMPCI,       { "JUMPCI",              2,    0, Tier::High } },
 }; 
  
 InstructionInfo instructionInfo(Instruction _inst)
 {
 	auto it = c_instructionInfo.find(_inst);
 	if (it != c_instructionInfo.end())
 		return it->second;
-	return InstructionInfo{{}, 0, 0, 0, Tier::Invalid};
+	return InstructionInfo{{}, 0, 0, Tier::Invalid};
```

Comment:
If you have all fields initialized in the declaration of `InstructionInfo`, you can just do `return InstructionInfo{}` or even `return {}`.